orion/pkg/repos/semester_repo_test.go

```diff
@@ -65,9 +106,7 @@ func TestSelectSemester(t *testing.T) {
 	defer db.Close()
 
 	// Mock DB statements and execute
-	now := time.Now().UTC()
-	rows := sqlmock.NewRows([]string{"Id", "CreatedAt", "UpdatedAt", "DeletedAt", "SemesterId", "Title"}).
-		AddRow(1, now, now, sql.NullTime{}, "2020_fall", "Fall 2020")
+	rows := getSemesterRows()
```

Comment:
I don't know if you if need a separate method for everything...

Reply:
If I ever change the semester, I would need to change it in 6 other places.